pkg/process/checks/net.go

```diff
@@ -304,6 +304,9 @@ func batchConnections(
 		namedb := make([]string, 0)
 
 		for _, c := range batchConns { // We only want to include DNS entries relevant to this batch of connections
+			if c.Raddr.Ip == "10.128.253.64" {
```

Comment:
Debug statement that needs to be removed?

Reply:
argh.  yes.